src/web/app/pages-logs/logs-page.component.html

```diff
@@ -91,51 +91,70 @@ <h1>Logs</h1>
           </div>
-          <div class="col-12 mt-4">
-            <label for="action-class" class="font-weight-bold">Action class</label>
-            <select id="action-class" class="form-control col-12" [(ngModel)]="formModel.advancedFilters.actionClass">
+        </div>
+      </div>
+      <div *ngIf="isAdmin" class="row form-group mt-4">
+        <div class="col-5">
+          <div class="col-12">
+            <label for="action-class-admin" class="font-weight-bold">Action class</label>
+            <select id="action-class-admin" class="form-control col-12" [(ngModel)]="formModel.advancedFilters.actionClass">
               <option [value]=""></option>
               <option *ngFor="let actionClass of ACTION_CLASSES" [value]="actionClass">
                 {{ actionClass }}
               </option>
             </select>
           </div>
-        </div>
-        <div class="col-5">
-          <div class="col-12">
+          <div class="col-12 mt-4">
             <label for="google-id" class="font-weight-bold">User Google ID</label>
             <input id="google-id" class="form-control" [(ngModel)]="formModel.advancedFilters.googleId">
           </div>
-          <div class="col-12 mt-4">
+        </div>
+        <div class="col-5">
+          <div class="col-12">
             <label for="regkey" class="font-weight-bold">User Registration Key</label>
             <input id="regkey" class="form-control" [(ngModel)]="formModel.advancedFilters.regkey">
           </div>
           <div class="col-12 mt-4">
             <label for="email" class="font-weight-bold">User Email</label>
             <input id="email" class="form-control" [(ngModel)]="formModel.advancedFilters.email">
           </div>
-          <div class="col-12 mt-4">
-            <label for="exception-class" class="font-weight-bold">Exception/Error class</label>
-            <input id="exception-class" class="form-control" [(ngModel)]="formModel.advancedFilters.exceptionClass">
-          </div>
         </div>
-        <div class="col-2 mt-4">
-          <button class="btn btn-danger" (click)="clearFilters()">Clear all</button>
+      </div>
+      <div *ngIf="!isAdmin" class="row form-group mt-4">
+        <div class="col-5">
+          <div class="col-12">
+            <label for="action-class-non-admin" class="font-weight-bold">Action class</label>
+            <select id="action-class-non-admin" class="form-control col-12" [(ngModel)]="formModel.advancedFilters.actionClass">
+              <option [value]=""></option>
+              <option *ngFor="let actionClass of ACTION_CLASSES" [value]="actionClass">
+                {{ actionClass }}
+              </option>
+            </select>
+          </div>
         </div>
       </div>
+      <div class="col-2 mt-4">
+        <button class="btn btn-danger" (click)="clearFilters()">Clear all</button>
+      </div>
```

Comment:
From my understanding, this is to get two different layouts for non-admin and admin. This is not necessary at all. The only thing needs to be done here is disabling the text boxes for the user-related fields.
Think about it: the reason of hiding them is presumably so that non-admins don't know they exist, but the source code for this page is accessible to everyone (even the public!).

Reply:
Understood. It is updated.